op-chain-ops/cmd/check-canyon/main.go

```diff
@@ -31,7 +31,7 @@ func CalcBaseFee(parent eth.BlockInfo, elasticity uint64, canyonActive bool) *bi
 	parentGasTarget := parent.GasLimit() / elasticity
 	// If the parent gasUsed is the same as the target, the baseFee remains unchanged.
 	if parent.GasUsed() == parentGasTarget {
-		return new(big.Int).Set(parent.BaseFee())
+		return new(big.Int).Set(parent.Basefee())
```

Comment:
Why the diff from `BaseFee` to `Basefee`?

Reply:
We use "Basefee" more consistently in the spec, and occasionally in the code, so just trying to improve consistency around this not only between code & spec but within the code itself.